### ISSUE TITLE
Missing some files into dist of v0.19.0

### ISSUE THREAD

**Issue by Chklang** [excaliburjs/Excalibur#1057]
I've added Excalibur to an empty application Angular7 application, and the "npm start" say :

```
ERROR in ./node_modules/excalibur/dist/Loader.js
Module not found: Error: Can't resolve './Loader.css' in '[...]\node_modules\excalibur\dist'
ERROR in ./node_modules/excalibur/dist/Loader.js
Module not found: Error: Can't resolve './Loader.logo.png' in '[...]\node_modules\excalibur\dist'
```

I've added these files from the Github repo and it's works. So can you add these files into next version?

Greats!

**Comment by eonarheim** [comment]
@chklang good catch, the npm pack must have missed some files. We'll issue a point release to fix this as soon as we can.

**Comment by eonarheim** [comment]
@Chklang as a work around importing the whole package from `excalibur.js` should work. The missing css and image are cooked into the final webpack output. Tree shaking doesn't provide a lot of value yet in excalibur so this isn't the worst work around.

```typescript

import * as ex from 'excalibur'
...

```

**Comment by eonarheim** [comment]
@Chklang Correction this is the workaround

`import * as ex from 'excalibur/dist/excalibur';`

**Comment by eonarheim** [comment]
@Chklang Just released v0.19.1, should fix this issue! Thanks!

**Comment by Chklang** [comment]
Thanks or your rapidity! ;)
